IndexedDB/idbindex_keyPath.any.js

```diff
@@ -28,3 +28,49 @@ indexeddb_test(
     t.done();
   },
   `IDBIndex's keyPath attribute returns the same object.`);
+
+  indexeddb_test(
+  (t, db) => {
+    const store = db.createObjectStore('store', {autoIncrement: true});
+    store.createIndex('index', ['a']);
+
+    store.add({a: 1, b: 2, c: 3})
+  },
+  (t, db) => {
+    const tx = db.transaction('store', 'readonly', {durability: 'relaxed'});
+    const store = tx.objectStore('store');
+    const index = store.index('index');
+    const cursorReq = index.openCursor();
+
+    cursorReq.onsuccess = t.step_func((e) => {
```

Comment:
Optional: this could use `t.step_func_done(...)` and skip the explicit `t.done()` call later on. I think most tests in this directory do use an explicit `t.done()` though.